doc/source/calacs_hstcal.rst

```diff
@@ -121,8 +138,8 @@ For post-SM4 full-frame WFC exposures, it also includes:
 Pixel-Based CTE Correction (PCTECORR)
 =====================================
 
-For all full-frame WFC exposures, pixel-based CTE correction (ACS ISR 2010-03
-and 2012-03) is applied in ACSCTE that occurs after the ACSCCD series;
```

Comment:
We should keep the old ISRs in case user somehow desires Gen 1 correction. Just make it clear that they are for Gen 1.